Patch to correct where STATIC_ROOT is set.  This has been fixed
upstream under https://bugs.launchpad.net/horizon/+bug/1399585 but not
yet backported to Kilo.

--- horizon-2015.1.0/openstack_dashboard/settings.py.orig 2015-07-30 13:43:47.751455772 -0600
+++ horizon-2015.1.0/openstack_dashboard/settings.py 2015-07-30 13:45:34.900930381 -0600
@@ -258,6 +258,10 @@ ADD_INSTALLED_APPS = []
 # It can be overridden in local_settings.py
 CUSTOM_THEME_PATH = 'static/themes/default'

+# Set these before importing local_settings in case they are set there
+MEDIA_ROOT = os.path.abspath(os.path.join(ROOT_PATH, '..', 'media'))
+STATIC_ROOT = os.path.abspath(os.path.join(ROOT_PATH, '..', 'static'))
+
 try:
     from local.local_settings import *  # noqa
 except ImportError:
@@ -272,9 +276,7 @@ if LOGOUT_URL is None:
 if LOGIN_REDIRECT_URL is None:
     LOGIN_REDIRECT_URL = WEBROOT

-MEDIA_ROOT = os.path.abspath(os.path.join(ROOT_PATH, '..', 'media'))
 MEDIA_URL = WEBROOT + 'media/'
-STATIC_ROOT = os.path.abspath(os.path.join(ROOT_PATH, '..', 'static'))
 STATIC_URL = WEBROOT + 'static/'
 STATICFILES_DIRS = get_staticfiles_dirs(WEBROOT)

